kubernetes/e2e_test/test_utils.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from kubernetes import utils, client
+from kubernetes.e2e_test import base
+
+class TestUtils(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.config = base.get_e2e_configuration()
+
+    def test_app_yaml(self):
+        k8s_api = utils.create_from_yaml(
+            "kubernetes/e2e_test/test_yaml/app.yaml", 
+            configuration=self.config)
+        self.assertEqual("apps/v1beta1", 
+            k8s_api.get_api_resources().group_version)
+        deployments = k8s_api.list_namespaced_deployment(
+            namespace="default").items
```

Comment:
You could use GET instead of LIST to test the creation: https://github.com/kubernetes-client/python/blob/529c0d3c2fbceca0f38d1cd3ae2eb3d5cd660a07/kubernetes/client/apis/apps_v1beta1_api.py#L3091